docs/src/components/Utils/ApiTable/ApiTable.component.tsx

```diff
@@ -93,7 +110,13 @@ const Table = styled('table')`
         }
 
         pre {
-            ${InlineCodeStyle};
+            display: inline;
+            background-color: rgba(27, 31, 35, 0.05);
+            font-family: 'SFMono-Regular', Consolas, Liberation Mono, Menlo,
```

Comment:
We should hoist this into the theme as a `mono` font, but this may not be the right time. @brandonlilly what are your thoughts?

Reply:
I like it 👍

Reply:
I like it 👍

Reply:
There are two options, haha: (1) hoist this into the theme or (2) leave it here and perform that work later.

Reply:
Added.